src/main/java/com/coveros/selenified/utilities/TestCase.java

```diff
@@ -80,6 +80,7 @@ public static String removeNonWordCharacters(String value) {
      * @return String: a unique name
      */
     public static String getTestName(Method method, Object... dataProvider) {
+        //TODO - look at making use of new setTestName and getFactoryParameters
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)